flight_phase_files/HALO_RF10_20200207_info.yaml

```
@@ -165,7 +166,8 @@ segments:
   start: 2020-02-07 17:14:50
   end: 2020-02-07 18:15:53
   segment_id: HALO-0207_c4
-  irregularities: [traffic_diversion, begin is at first launch due to roll angle at start]
+  irregularities:
+  - traffic_diversion, begin is at first launch due to roll angle at start
```

This would change from two individual irregularities to one with a comma in between. I think this would be better.
```suggestion
  - traffic_diversion
  - begin is at first launch due to roll angle at start
```